tests/tests/fv3_ccpp_gsd

```diff
@@ -118,6 +118,13 @@ export IMFSHALCNV=3
 export IMFDEEPCNV=3
 export LSM=3
 export LSOIL_LSM=9
+export DO_UGWP_V0=.F.
```

Comment:
Lines 121-127 should also be added to `tests/default_vars.sh` in the appropriate place (look for the gravity wave drag options). Then they shouldn't be required here; only in the drag suite tests that change the default values.